pkgs/applications/networking/mailreaders/claws-mail/default.nix

```diff
@@ -80,8 +82,16 @@ stdenv.mkDerivation {
     ++ optional (!enablePluginVcalendar) "--disable-vcalendar-plugin"
     ++ optional (!enableSpellcheck) "--disable-enchant";
 
+  enableParallelBuilding = true;
+
   postInstall = ''
     mkdir -p $out/share/applications
     cp claws-mail.desktop $out/share/applications
+
+    if test -e $out/share/mime; then
```

Comment:
This is me being paranoid, because if for some reason Claws gets an `$out/share/mime`, then the `ln -s` will end up creating `$out/share/mime/mime -> ...`.